apps/meteor/tests/end-to-end/api/livechat/methods/changeLivechatStatus.ts

```diff
@@ -0,0 +1,226 @@
+/* eslint-env mocha */
+
+import type { ILivechatAgent, IUser } from '@rocket.chat/core-typings';
+import type { Response } from 'supertest';
+import { expect } from 'chai';
+
+import { getCredentials, request, credentials, methodCall } from '../../../../data/api-data';
+import { createAgent } from '../../../../data/livechat/rooms';
+import { updatePermission, updateSetting } from '../../../../data/permissions.helper';
+import { password } from '../../../../data/user';
+import { createUser, getMe, login } from '../../../../data/users.helper';
+import { disableDefaultBusinessHour, makeDefaultBusinessHourActiveAndClosed } from '../../../../data/livechat/businessHours';
+
+describe('livechat:changeLivechatStatus', function () {
```

Comment:
I don't think it is bad having more tests, but I got a bit surprised having new tests for a method that is being deprecated 🙈

Reply:
😬 

let's say they'll be there until the method is removed :trollface: